tests/integrations/tests/integrations/test_bedrock.py

```
@@ -0,0 +1,300 @@
+                        header, data = url.split(",", 1)
+                        media_type = header.split(";")[0].split(":")[1]
+                        image_bytes = base64.b64decode(data)
+                        content.append({
+                            "image": {
+                                "format": media_type.split("/")[1],  # png, jpeg
+                                "source": {"bytes": image_bytes}
+                            }
+                        })
+                    else:
+                        # URL image - download and convert to bytes
+                        response = requests.get(url)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)